config/initializers/feature_decisions.rb

```diff
@@ -75,3 +75,6 @@
 OpenProject::FeatureDecisions.add :change_hierarchy_item_parent,
                                   description: "Enables a functionality to change the parent of a hierarchy item of " \
                                                "custom fields of type hierarchy and scored list."
+
+OpenProject::FeatureDecisions.add :new_project_overview,
+                                  description: "Enables the new project overview experience."
```

Comment:
Not sure about this description - sounds a tad Microsoft-ey.

Reply:
Haha, indeed 😅  But I wouldn't care too much about it, as mostly devs will read that.